smoke-test/tests/cypress/integration_test.py

```diff
@@ -17,7 +17,7 @@ def ingest_cleanup_data():
 
 
 def test_run_cypress(frontend_session, wait_for_healthchecks):
-    command = f"npx cypress run --record"
```

Comment:
Why are we removing the Cypress recording?

Reply:
cc: @eclaassen-pb , @gabe-lyons could one of you respond to this question?

Reply:
I would not remove the cypress recording.

Reply:
I was advised to by @aditya-radhakrishnan because it was failing on the datahub-fork project with

```
Opening Cypress...
Your Record Key <redacted> is not valid with this projectId: <redacted>

It may have been recently revoked by you or another user.

Please log into the Dashboard to see the valid record keys.
```

Reply:
Got it, yeah since this is the OSS PR let's roll this back and only have the other changes 😄

Reply:
Agree @RyanHolstien! Rolling back this change.

Reply:
Reverted the change, @RyanHolstien!

Reply:
Thanks!